deploy/cloudformation/manifest-pipeline-pipeline.yml

```diff
@@ -260,6 +263,7 @@ Resources:
             Resource:
               - !Sub 'arn:aws:lambda:${AWS::Region}:${AWS::AccountId}:function:${TestStackName}-*'
               - !Sub 'arn:aws:lambda:${AWS::Region}:${AWS::AccountId}:function:${ProdStackName}-*'
+              - !Sub 'arn:aws:lambda:${AWS::Region}:${AWS::AccountId}:layer:*'
```

Comment:
Can you refine this permission? Looks like you can do a namespace pattern similar to the others https://docs.aws.amazon.com/lambda/latest/dg/access-control-identity-based.html#permissions-user-layer

Reply:
Sure it'll have to be a little different since layer stack names can be different than the pipeline stack names.